integration/rpctest/memwallet.go

```diff
@@ -125,7 +125,7 @@ func newMemWallet(net *chaincfg.Params, harnessID uint32) (*memWallet, error) {
 
 	// The first child key from the hd root is reserved as the coinbase
 	// generation address.
-	coinbaseChild, err := hdRoot.Child(0)
+	coinbaseChild, err := hdRoot.Derive(0)
```

Comment:
Just wanted to note that this is fine since it's just an in-memory wallet, but otherwise we would've needed to use the "non standard" version.